vj4/ui/templates/domain_main.html

```diff
@@ -126,32 +126,31 @@ <h1 class="section__title">{{ _('Discussion') }}</h1>
   {% if domain_id == vj4.model.builtin.DOMAIN_ID_SYSTEM %}
     <div class="section side">
       <div class="section__header">
-        <h1 class="section__title">{{ _('Recommanded') }}</h1>
+        <h1 class="section__title">{{ _('Recommended') }}</h1>
       </div>
       <div class="section__body"><ul class="group-list">
         <li class="group-list__item">
           <h2 class="section__title">{{ _('Chinese') }}</h2>
           <ol class="chip-list">
-            <li class="chip-list__item"><a class="typo-a" href="https://www.xuetangx.com/courses/course-v1:TsinghuaX+30240184+sp/about" target="_blank">清华 MOOC (上)</a></li>
-            <li class="chip-list__item"><a class="typo-a" href="https://www.xuetangx.com/courses/course-v1:TsinghuaX+30240184_2X+sp/about" target="_blank">清华 MOOC (下)</a></li>
+            <li class="chip-list__item"><a class="typo-a" href="https://www.xuetangx.com/courses/course-v1:TsinghuaX+30240184+sp/about" target="_blank">清华数据结构 MOOC</a></li>
             <li class="chip-list__item"><a class="typo-a" href="https://www.51nod.com/" target="_blank">51Nod</a></li>
             <li class="chip-list__item"><a class="typo-a" href="https://www.luogu.org/" target="_blank">洛谷</a></li>
             <li class="chip-list__item"><a class="typo-a" href="https://www.rqnoj.cn/" target="_blank">RQNOJ</a></li>
-            <li class="chip-list__item"><a class="typo-a" href="http://www.lydsy.com/JudgeOnline/" target="_blank">BZOJ {{ _('(unsafe)') }}</a></li>
```

Comment:
HTTPS is a nice-to-have feature, not a critical one. Thus I think we should not output our values of "security" to the user, which is out of our scope. We can provide a screen to notice user that he/she is leaving our website and add a hint that the website is insecure though. However this should be applied to all external links on our website.